tests/integration/test_integration.py

```diff
@@ -771,6 +771,21 @@ def test_ddtrace_run_startup_logging_injection(ddtrace_run_python_code_in_subpro
     assert b"ValueError: Formatting field not found in record: 'dd.service'" not in err
 
 
+def test_no_module_debug_log(ddtrace_run_python_code_in_subprocess):
+    env = os.environ.copy()
+    env.update(
+        dict(
+            DD_TRACE_DEBUG="1",
+            DD_CALL_BASIC_CONFIG="1",
+        )
+    )
+    out, _, _, _ = ddtrace_run_python_code_in_subprocess(
```

Comment:
We are planning to remove DD_CALL_BASIC_CONFIG in a future release so it might be best to not to use in a future tests.

Would something like this work instead:

```
code = """import logging
from ddtrace import patch

logging.basicConfig(level=logging.DEBUG)
patch(starlette=True)
"""

out, _, _, _ = ddtrace_run_python_code_in_subprocess(code)
assert b"DEBUG:ddtrace._monkey:integration starlette not available (missing module: starlette)" in out
```

Reply:
sure